clientd3d/list.c

```diff
@@ -328,6 +347,85 @@ list_type list_move_to_front(list_type l, void *data, CompareProc compare)
    return l;
 }
 /************************************************************************/
+/*
+ * list_move_to_nth:  Find the given item in the given list, and
+ *   move it to the nth place.
+ *   Returns the list.
```

Comment:
This comment doesn't match what the function does.  For example, there's no argument "n".

Reply:
Renamed function to list_move_item and completely redid all the comments in a new commit.